.travis.yml

```diff
@@ -11,10 +11,13 @@ go:
  - master
 
 env:
- - TAGS=""
- - TAGS="-tags bounds"
- - TAGS="-tags noasm"
- - TAGS="-tags appengine"
+ global:
+  - GO111MODULE=on
```

Comment:
this enables the "module-mode" for all variants.
is it really what we want to do?

I'd argue that `GO111MODULE=on` should be set:
- for `master` (it will be the default for 1.13, eventually)
- for a matrix entry for 1.12

otherwise, we might get some impedance mismatch from users who are still using `GOPATH`+`GO111MODULE=auto`.

Reply:
>this enables the "module-mode" for all variants.
> is it really what we want to do?

Yes. It's ignored for go1.10 and tests that go1.11 and go1.12 use modules correctly.

> otherwise, we might get some impedance mismatch from users who are still using `GOPATH`+`GO111MODULE=auto`.

I don't see a failure mode in that direction. Can you given me an example of how that might fail in a meaningful way for a user? The only failures that I can see happening when not in module mode are when the user does not have an adequately recent version of golang.org/x/exp/rand, and that is only a test failure, not a package failure. We don't import anything outside gonum.org/v1/gonum except stdlib and golang.org/x/exp/rand.